packages/jupyter-chat/src/footers/types.ts

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright (c) Jupyter Development Team.
+ * Distributed under the terms of the Modified BSD License.
+ */
+
+import { IChatModel } from '../model';
+import { IChatMessage } from '../types';
+
+/**
+ * The props sent passed to each `MessageFooterSection` React component.
+ */
+export type MessageFooterSectionProps = {
+  model: IChatModel;
```

Comment:
Maybe we should send only the `IChatContext` here, in which we could add some `ChatModel` signals. That way the component could be reactive the changes in the model without accessing the whole methods of the model.